README.rst

```diff
@@ -67,17 +127,18 @@ By default, autoapi uses a local SQLite database. To specify a different databas
 
 For details on RDS services available through 18F Cloud Foundry, see https://docs.cloud.gov/apps/databases/.
 
-Example Implementation
-----------------------
-
-* https://autoapi.18f.gov/
-* if you want to see json, you can `curl https://autoapi.18f.gov`
-* https://autoapi.18f.gov/admin
```

Comment:
With the exception of the `/admin` endpoint, this seems like a repeat of the information in the "Example Instance" section at the top of the readme, so I removed it.  As for the `/admin` endpoint, I removed that because it's asking for a username/password and the user won't know what that is--speaking of which,  we should probably document how to set that up, right?